aiohttp/payload.py

```diff
@@ -150,6 +151,11 @@ def __init__(self, value, *args, **kwargs):
 
         self._size = len(value)
 
+        if self._size > DEFAULT_LIMIT:
```

Comment:
ok, do you want it in `payload.py` or in `stream.py` next to the other constants?